lib/podium.ex

```diff
@@ -28,38 +28,37 @@ defmodule Podium do
       |> inject_application_uid()
       |> inject_source_type()
 
-    conversation_item =
-      Caramelize.camelize(%{
-        conversation_item: conversation_item
-      })
+    params = %{
+      conversation_item: conversation_item
+    }
 
-    API.post("/conversation_items", Caramelize.camelize(conversation_item))
+    API.post("/conversation_items", Caramelize.camelize(params))
   end
 
   @doc """
   Update a [conversation item](https://hexdocs.pm/podium_ex/Podium.ConversationItem.html#content).
   """
   @spec update_conversation_item(ConversationItem.t()) :: ConversationItem.t()
-  def update_conversation_item(%ConversationItem{uid: uid} = item) do
+  def update_conversation_item(%ConversationItem{} = item) do
     conversation_item =
       item
       |> remove_nils()
       |> inject_application_uid()
       |> inject_source_type()
 
-    conversation_item = %{
+    params = %{
       conversation_item: conversation_item
     }
 
-    API.put("/conversation_items/#{uid}", Caramelize.camelize(conversation_item))
+    API.put("/conversation_items", Caramelize.camelize(params))
```

Comment:
Can we undo all of these changes? Or do updates really not use the `uid` in the URL anymore?

Reply:
It will not work if you include the `uid` in the URL.